recipes/gsi-baryon/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "gsi-baryon" %}
+{% set version = "1.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gsitechorg/fermion/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5af20717722fcdcdcb6f613b9e9e5afc1b3f6277be109354e36b73cd909b4c88
+
+build:
+  number: 0
+  skip: true  # [not unix]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake =3.26.4
+    - gtest =1.13.0
+    - make =4.3
+    - open-belex-libs >=1.0.1,<2.0.0
+    - open-belex >=1.0.1,<2.0.0
+    - python =3.11
+    - rapidcheck =2023.4.13
+    - yaml-cpp =0.7.0
```

Except make and cmake, everything should be in host and the run section should only contain python
```suggestion
    - cmake =3.26.4
    - gtest =1.13.0
    - make =4.3
    - open-belex-libs >=1.0.1,<2.0.0
    - open-belex >=1.0.1,<2.0.0
    - python =3.11
    - rapidcheck =2023.4.13
    - yaml-cpp =0.7.0
```

---

I don't quite understand the purpose of `host` dependencies, how are they different from the `build` and `run` dependencies? Also, `open-belex`, `open-belex-libs`, and `python` are only used during the compilation process to generate the respective libraries. Since they are not runtime dependencies, should they still be moved from `build` to `host`?

---

`build` is for compiler, `host` for dependencies you build against and `run` is what you need at runtime.